### DISCUSSION TITLE
Opening other bunches

### DISCUSSION THREAD

**Discussion by andytough** [ttscoff/bunch-docs#332] [Q&A]
I may be wrong, but I think that if you call another .bunch file from the .bunch file you are opening, it does not launch the other bunch if it is the first entry in the .bunch file you are opening.

So this successfully opens web-dev-static-sites.bunch...

File name: web-dev-lse-accessibility.bunch

```
---
title: LSE Accessibility
---
%Adobe InDesign 2024
%Finder
- ~/Data/clients/lse_local/lseaccessibility/
safari:http://0.0.0.0/
web-dev-static-sites.bunch
```

This does not open web-dev-static-sites.bunch...
```
---
title: LSE Accessibility
---
web-dev-static-sites.bunch
%Adobe InDesign 2024
%Finder
- ~/Data/clients/lse_local/lseaccessibility/
safari:http://0.0.0.0/
```

Just in case it is relevant here is the content of web-dev-static-sites.bunch

File name: web-dev-static-sites.bunch

```
---
title: Web dev - static sites
---
%Typora
%Fork
%Warp
%BBEdit
```

I am using macOS Sonoma 14.5 on a Macbook Pro M1.

If this is expected behaviour it might be worth mentioning in the documentation.

Thank you for making Bunch Brett. I have been meaning to make use of it for years but never got around to it. Now I have tried it I am finding it really useful.

**Comment by ttscoff** [comment]
Interesting parsing error that I'll have to look into. Only seems to 
happen if there's no line break and the next line starts with %. If you 
add a blank line before `%Adobe...` or run it without the `%`, it works 
fine. Will look into it.

-Brett

**Reply by andytough** [reply]
Thank you.
